easybuild/easyconfigs/r/ReFrame/ReFrame-3.3.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'PythonBundle'
+
+name = 'ReFrame'
+version = '3.3'
+
+homepage = 'https://github.com/eth-cscs/reframe'
+description = '''ReFrame is a framework for writing regression tests for HPC systems.'''
+
+toolchain = SYSTEM
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+req_py_majver = 3
+req_py_minver = 6
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+use_pip = True
+
+exts_list = [
+    ('pip', '20.2.4', {
+        'use_pip': False,
+        'checksums': ['85c99a857ea0fb0aedf23833d9be5c40cf253fe24443f0829c7b472e23c364a1'],
+    }),
+
+    # Add a setuptools version supporting the `find_namespace_packages` function
+    ('setuptools', '50.3.2', {
+        'source_tmpl': 'setuptools-%(version)s.zip',
+        'checksums': ['ed0519d27a243843b05d82a5e9d01b0b083d9934eaa3d02779a23da18077bd3c'],
+    }),
+    ('reframe', version, {
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/eth-cscs/reframe/archive/'],
+        'checksums': ['9da150a051e9fa4ffea1361f30e8593261e7f6ebc71ec91ed32143539f871ad7'],
+    }),
+]
+
+postinstallcmds = [
+    # Use bootstrap script to install ReFrame dependencies under `external`.
+    # ReFrame adds `external` to its own python path before executing, without
+    # affecting the python path of the system
+    './bootstrap.sh +docs +pygelf', 'cp -r external %(installdir)s',
```

Comment:
@teojgo Should this be done via `preinstallopts` in the `reframe` extension above, to avoid the installation problems popping up in the test reports (the installation of `reframe` fails because `jsonschema` is not available)?